crates/swc_ecma_minifier/tests/exec.rs

```diff
@@ -11325,3 +11325,13 @@ fn issue_9010() {
         "#,
     );
 }
+
+#[test]
+fn issue_9184() {
+    run_default_exec_test(
+        r#"
+        let pi= Math.random() >1.1 ? "foo": "bar";
```

Comment:
Can we have one more case for the reverse case? (`Math.random() < -1`)

Reply:
I am not quite understand why adding the reverse case.
IMO, the key of this case is about  purify the nested tpl string. 
The `Math.random()` is just for the `pi` variable not being to replaced by its literal value in the tpl string.

Reply:
Ah, ok